Sources/Collections/Observable2DArray.swift

```diff
@@ -377,6 +377,10 @@ extension Observable2DArrayEvent: DataSourceEventProtocol {
 }
 
 extension Observable2DArray: DataSourceProtocol {
+
```

Comment:
Make `Observable2DArray` conform to `QueryableDataSourceProtocol` instead. Index type would be `IndexPath`.